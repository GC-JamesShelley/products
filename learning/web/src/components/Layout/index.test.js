import React from "react"
import Layout from "./index"
import { shallow } from "enzyme"

const location = { pathname: "test" }

describe(Layout, () => {
  it("should render", () => {
    const component = shallow(
      <Layout
        location={location}
        title={"Hola"}
        children={<></>}
        withSidebar={true}
      />
    )
    expect(component).toMatchSnapshot()
  })
})
