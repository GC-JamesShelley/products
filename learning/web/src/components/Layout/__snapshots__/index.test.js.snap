// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render üç™ 1`] = `
<styled.div>
  <CookieBanner />
  <styled.header>
    <styled.div>
      <HeaderLogoLink />
      <h1>
        Hola
      </h1>
    </styled.div>
  </styled.header>
  <styled.main>
    <styled.div>
      <styled.div>
        <Sidebar
          location={
            Object {
              "pathname": "test",
            }
          }
        />
        <styled.div />
      </styled.div>
    </styled.div>
  </styled.main>
  <styled.footer>
    <styled.div>
      <styled.picture>
        <SvgAgencyDigitalLogo />
      </styled.picture>
      <styled.nav>
        <ul>
          <li>
            <p>
              <mockConstructor
                to="/cookies"
              >
                Cookie Policy
              </mockConstructor>
            </p>
          </li>
          <li>
            <p>
              <a
                href="https://www.gov.uk/government/publications/mhra-privacy-notice/mhra-privacy-notice"
                rel="noopener noreferrer"
                target="_blank"
              >
                Privacy Policy
              </a>
            </p>
          </li>
          <li>
            <p>
              <mockConstructor
                to="/accessibility"
              >
                Accessibility Statement
              </mockConstructor>
            </p>
          </li>
        </ul>
      </styled.nav>
    </styled.div>
  </styled.footer>
</styled.div>
`;
